### PR TITLE
Updated dependencies and use CSP-safe build of acorn

### PR DESCRIPTION
**Pull request by call-a3**
The title speaks for itself I think. Ran the tests, all pass.

### CODE REVIEW

**Comment by ForbesLindesay** [comment]
I'm not sure we should be defaulting to the csp version.  If I remember correctly it performs worse, and it is not the default.

**Comment by call-a3** [comment]
Normally I would agree with you, (for the sake of performance) but CSP in some common contexts (Travis-CI being the first and foremost I can think of) makes this library and any (indirect) dependants unusable in those environments... I think, for the sake of cross-environment, the (small) performance hit is worth the added benefit of supporting those envs.

**Comment by call-a3** [comment]
Another option would be to extrapolate the approach of acorn and provide two "interfaces" of falafel, one that uses the faster (default) acorn, and one that is CSP-proof and uses acorn_csp

**Comment by call-a3** [comment]
@ForbesLindesay Any further comments on this? Things I should change?

**Comment by ForbesLindesay** [comment]
Looks good to me!

Unfortunately I don't think I have permission to publish on npm.

@substack could you publish this?  And perhaps add `forbeslindesay` as an owner.

**Comment by substack** [comment]
Why does this patch reformat everything? These changes are far too monolithic.

**Comment by substack** [comment]
I've reverted these changes because they change too much at once and add too many extraneous features.

**Comment by substack** [comment]
Sorry for the harshness but I really think each of the elements of this pull request should be handled separately. This pull request touches on:

* indentation level formatting
* code structure
* library upgrades
* addition of badges

It's really hard to follow the progress and implications of a pull request if so many issues are handled at once and this kind of thing causes far too much thrashing.

**Comment by call-a3** [comment]
Fair enough. The change in formatting was unintentional, so I'll try to skip that when I make new pull requests.
I'll split these changes up into different parts.
